common/utilities/integration_adapters_logger.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+import time
+
+AUDIT = 25
+
+
+def _add_audit_level_log():
+    """
+    Adds a new log level to the standard python logging module, this level is the `audit` level and is defined as
+    a high level than INFO but lower than  WARNING
+    """
+    logging.addLevelName(AUDIT, "AUDIT")
+
+    def audit(self, message, *args, **kws):
+        if self.isEnabledFor(AUDIT):
+            self._log(AUDIT, message, args, **kws)
+
+    logging.Logger.audit = audit
```

Comment:
Possibly better to use [`getLoggerClass`](https://docs.python.org/3.7/library/logging.html#logging.getLoggerClass):
```suggestion
    logging.getLoggerClass().audit = audit
```

Reply:
This has been removed anyway